src/name.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * Create a enhancer to add an name property to the store lifted by an enhancer.
+ * @param   {string}     name   the name to apply to the store.
+ * @returns {Function} An enhancer capable of lifting an enhancer.
+ */
+export function name(name) {
+  return enhancer => next => (reducer, initialState) => {
+    return {
+      ...enhancer(next)(reducer, initialState),
+      name,
+    };
+  };
+}
+
+/**
+ * Create a enhancer to add an name property to the store lifted by an enhancer.
+ * The base store will always be available under the `default` property.
+ * @param   {[type]}   store   a redux store.
```

Comment:
Missing `type` and spacing is off. I think existing documentation has no funny "alignment".